roles/content_view_version_cleanup/tasks/main.yml

```diff
@@ -0,0 +1,29 @@
+---
+- name: find all CVs
```

Comment:
For the name, since that is the part that appears on screen to a user, I would prefer to spell out the names of things. I'm OK if we use abbreviations in the code.

Reply:
+1